packages/kbn-legacy-logging/tsconfig.json

```diff
@@ -1,19 +1,14 @@
 {
   "extends": "../../tsconfig.base.json",
   "compilerOptions": {
-    "incremental": false,
+    "incremental": true,
     "outDir": "target",
     "stripInternal": false,
     "declaration": true,
     "declarationMap": true,
     "sourceMap": true,
```

Comment:
@jbudz I forgot. Can we add `rootDir: "src"` here?

Reply:
Pushed with 6099717